Version 1.1.2

Base29 opened the pull request:
* Fixed count issue on parent category.
* Fixed page title issue.
* Modifications/edits from Andrew

Base29 commented:
please don't merge it yet ...

Base29 commented:
you can merge this now

EnigmaWeb commented:
Please can you check the readme, the bulk of which appears to be an old version. I think you'll have to update the readme in your branch manually so you don't overwrite the proper readme in master branch. Thanks.

Base29 commented:
readme should be Identically with the master branch because before working on the plugin I have pulled the changes done in the master branch and merged those in to the development branch ... so the readme should be merged as well .... I will check this

Base29 commented:
some how you don't have this in the readme in the master branch ... 

```
== Changelog ==
= 1.1.2 =
* Fixed count issue on parent category.
* Fixed page title issue.

= 1.1.1 =
* fixed warnings.
* 'parent' => 0 removed from the terms array

= 1.1 =
* Fixed search template issue.
* Fixed widgets issue 
* Fixed template redirect issue.
* Some minor css issues fixed.
```

Base29 commented:
it seems that you are using the old readme .... ??

EnigmaWeb commented:
That's what I'm saying, they've got out of sync.  Master branch is missing the new changelog stuff obviously because it's not been merged yet, but Dev branch is missing a large amount of other stuff... check the dif you'll see...

EnigmaWeb commented:
Oh doesn't really matter, I can merge that file manually. 
I'm really raising it because it indicates a workflow problem. I think we need to start closing Dev branches after they've been merged so that when you start new work you can check out the new branch and be sure you've got all the latest stuff from master.

Base29 commented:
Sure … I will check what I can do … because as I said that the development branch is connected to my IDE and the development server …

EnigmaWeb commented:
Merged, and deleted branch. Dev branches need to be closed when we're done with them, except in specific circumstances. Faisal, we can discuss this more, but you might like to read up on git workflow.
